app/services/importer/todoist.rb

```diff
@@ -0,0 +1,35 @@
+require 'open-uri'
+require 'json'
+
+class Importer::Todoist
+  API_URL = 'https://todoist.com/API/v7/sync'
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.